easybuild/easyconfigs/r/ReFrame/ReFrame-3.0.eb

```diff
@@ -69,7 +69,4 @@ sanity_check_commands = ['reframe -V']
 
 sanity_pip_check = True
 
-# need to add 'bin' subdir to $PATH explicitly to ensure right 'pip' command is used for installing extensions
-modextrapaths = {'PATH': 'bin'}
```

Comment:
The comment here suggests we had a very good reason to include this `modextrapaths`.

There's an important difference between adding `bin` to `$PATH` unconditionally via `modextrapaths`, or letting EasyBuild do it when creating the module file *if a `bin` subdirectory exists*...

Reply:
To avoid totally ignoring the comment, we can add "`export PATH=%(installdir)s/bin:$PATH &&`" to `preinstallopts` for the `reframe` extensions, like I did in #14477

Reply:
fixed in 42e2ab8b54